modules/preview-react/segmented-control/README.md

```diff
@@ -0,0 +1,36 @@
+# Canvas Kit Segmented Control
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/preview-react/README.md">
+  <img src="https://img.shields.io/badge/PREVIEW-beta-blueviolet" alt="PREVIEW: Beta" />
+</a>  This component is work in progress and currently in prerelease.
+
+
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+```tsx
+import * as React from 'react';
+import {SegmentedControl} from '@workday/canvas-kit-preview-react/segmented-control';
+
+<SegmentedControl />;
+```
+
+## Static Properties
+
+> None
+
+## Component Props
+
+### Required
+
+> None
+
+### Optional
```

Comment:
Should this readme match what we usually do?
```# Canvas Kit Modal

A Modal component that allows you to render a Popup with translucent overlays.

## Installation

```sh
yarn add @workday/canvas-kit-react
```

For more detailed information on this component, please refer to the
[storybook documentation](https://workday.github.io/canvas-kit/?path=/docs/components-popups-modal-react)```

Reply:
updated. thanks for catching